docs/registries.conf.5.md

```diff
@@ -0,0 +1,28 @@
+% registries.conf(5) System-wide registry configuration file
+% Brent Baude
+% Aug 2017
+
+# NAME
+registry.conf - Syntax of System Registry Configuration File
+
+# DESCRIPTION
+The REGISTRIES configuration file is a system-wide configuration file for container image
+registries. The file format is YAML.
+
+# FORMAT
+The [YAML_format][yaml] is used to build simple list format for registries under two
+categories: `registries` and `insecure_registries`.
+
+The following example configuration defines two registries and one insecure registries.
+
+```
+registries:
+    - registry1.com
+    - registry2.com
+
+insecure_registries:
+    - registry3.com
+```
```

Comment:
FWIW.  I think you ought to hold off on your blog post then.  I'd hate to have the blog go out and then a month from now the file you talk about configuring not be valid....